majilove/models.py

```diff
@@ -106,3 +108,402 @@ def display_blood_type(self):
+
+    COLORS= OrderedDict([
+        (1, { # yellow
+            'translation': _('Star'),
+            'english': u'Star'
+        }),
+        (2, { # red
+            'translation': _('Shine'),
+            'english': u'Shine'
+        }),
+        (3, { # blue
+            'translation': _('Dream'),
+            'english': u'Dream'
+        })
+    ])
+
+    COLOR_CHOICES = [(_name, _info['translation']) for _name, _info in COLORS.items()]
+    COLOR_WITHOUT_I_CHOICES=True
+    i_color = models.PositiveIntegerField(_('Color'), choices=COLOR_CHOICES, db_index=True)
+    english_color = property(getInfoFromChoices('color', COLORS, 'english'))
+
+    STATS = OrderedDict([
+        (1, {
+            'translation': _('Dance'),
+            'english': u'Dance',
+            'japanese_translation': u'DANCE'
+        }),
+        (2, {
+            'translation': _('Vocal'),
+            'english': u'Vocal',
+            'japanese_translation': u'VOCAL'
+        }),
+        (3, {
+            'translation': _('Charm'),
+            'english': u'Charm',
+            'japanese_translation': u'ACT'
+        })
+    ])
+
+    dance_min = models.PositiveIntegerField(string_concat(_('Dance'), ' (', _('Minimum'), ')'), default=0)
+    dance_single_copy_max = models.PositiveIntegerField(string_concat(_('Dance'), ' (', _('Single Copy Maximum'), ')'), default=0)
+    dance_max_copy_max = models.PositiveIntegerField(string_concat(_('Dance'), ' (', _('Maxed Copy Maximum'), ')'), default=0)
+
+    vocal_min = models.PositiveIntegerField(string_concat(_('Vocal'), ' (', _('Minimum'), ')'), default=0)
+    vocal_single_copy_max = models.PositiveIntegerField(string_concat(_('Vocal'), ' (', _('Single Copy Maximum'), ')'), default=0)
+    vocal_max_copy_max = models.PositiveIntegerField(string_concat(_('Vocal'), ' (', _('Maxed Copy Maximum'), ')'), default=0)
+
+    charm_min = models.PositiveIntegerField(string_concat(_('Charm'), ' (', _('Minimum'), ')'), default=0)
+    charm_single_copy_max = models.PositiveIntegerField(string_concat(_('Charm'), ' (', _('Single Copy Maximum'), ')'), default=0)
+    charm_max_copy_max = models.PositiveIntegerField(string_concat(_('Charm'), ' (', _('Maxed Copy Maximum'), ')'), default=0)
+
+    @property
+    def overall_min(self):
+        return self.dance_min + self.vocal_min + self.charm_min
+
+    @property
+    def overall_single_copy_max(self):
+        return self.dance_single_copy_max + self.vocal_single_copy_max + self.charm_single_copy_max
+
+    @property
+    def overall_max_copy_max(self):
+        return self.dance_max_copy_max + self.vocal_max_copy_max + self.charm_max_copy_max
+
+    # Leader Skill
+    LEADER_SKILL_INFO = {
+        'variables': ['color', 'stat', 'percentage'],
+        'template': _(u'{color} {stat} +{percentage}%'),
+        'japanese_template': u'{color} の{stat}パフォーマンス{percentage}%上昇'
+    }
+
+    #currently always the same as color, but this is safer
```

Comment:
I'd recommend not storing it for now to avoid data inconsistency and maybe having an alias like so:

```python
@property
def leader_skill_color(self):
    return self.color
```

You can always go back in your code and change that to a proper field if needed.

You may also want to allow accessing the `i_` raw value with a similar property, but I don't see any case where that would be useful.